crates/core/src/operations/merge/mod.rs

```diff
@@ -60,6 +63,7 @@ use filter::try_construct_early_filter;
 use futures::future::BoxFuture;
 use parquet::file::properties::WriterProperties;
 use serde::Serialize;
+use tracing::field::debug;
 use tracing::log::*;
```

Comment:
`debug!` comes in on the second line, I'm guessing this was an IDE trying to be clever. I'll remove this in a later change so as not to block this.